packages/documentation/copy/en/developer/Downloading and Building.md

```diff
@@ -1,34 +1,47 @@
 ---
-title: Downloading
+title: Developer Setup
```

Comment:
The title here does not match the file name anymore...

Reply:
I was a bit of afraid renaming the file as I did not know how it is included in the build and found, but apparently a simple file rename just works.

Reply:
Or not... I don't know why the CI build fails now, it works fine locally